addons/web/static/src/js/components/action_menus.js

```diff
@@ -147,6 +147,7 @@ odoo.define('web.ActionMenus', function (require) {
                 action: result,
                 options: {
                     on_close: () => this.trigger('reload'),
+                    fullscreen: this.env.device.isMobile
```

Comment:
Watch out with that : there is still a task blocked about fullscreen modals on mobile (tldr: sometimes it's good, but not always...) and no consensus about it.